content/guides/translation-connection/introduction.md

```diff
@@ -0,0 +1,24 @@
+# Introduction
+![Asset family illustration](../../img/illustrations/illus--Translatedproduct.svg)
+
+> Do you want to connect your PIM to your online translation solution?
+> You want to implement a product translation feature and you don't know where to start?
+
+You landed in the right place!
+
+## Goal of this guide
+
+In the following guide, we are going to explain how to properly connect our PIM to your online Translation solution, thanks to our wonderful API.
```

Comment:
```suggestion
In the following guide, we are going to explain how to properly connect our PIM to your online translation solution, thanks to our wonderful API.
```
I'll let you remove the uppercase at all your "Translation solution" throughout this document.